# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/sendmail/setmailer.patch
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./sendmail/Makefile.m4.orig	2002-09-09 04:48:54.000000000 +0200
+++ ./sendmail/Makefile.m4	2003-03-11 16:30:07.000000000 +0100
@@ -2,7 +2,7 @@
 include(confBUILDTOOLSDIR`/M4/switch.m4')
 
 define(`confREQUIRE_LIBSM', `true')
-bldPRODUCT_START(`executable', `sendmail')
+bldPRODUCT_START(`executable', `sendmail_sendmail')
 define(`bldBIN_TYPE', `G')
 define(`bldINSTALL_DIR', `')
 define(`bldSOURCES', `main.c alias.c arpadate.c bf.c collect.c conf.c control.c convtime.c daemon.c deliver.c domain.c envelope.c err.c headers.c macro.c map.c mci.c milter.c mime.c parseaddr.c queue.c readcf.c recipient.c sasl.c savemail.c sfsasl.c shmticklib.c sm_resolve.c srvrsmtp.c stab.c stats.c sysexits.c timers.c tls.c trace.c udb.c usersmtp.c util.c version.c ')
@@ -14,7 +14,7 @@
 define(`bldTARGET_INST_DEP', ifdef(`confINST_DEP', `confINST_DEP',
 `${DESTDIR}/etc/mail/submit.cf ${DESTDIR}${MSPQ}'))dnl
 define(`bldTARGET_LINKS', ifdef(`confLINKS', `confLINKS',
-`${DESTDIR}${UBINDIR}/newaliases ${DESTDIR}${UBINDIR}/mailq ${DESTDIR}${UBINDIR}/hoststat ${DESTDIR}${UBINDIR}/purgestat')
+`${DESTDIR}${UBINDIR}/newaliases_sendmail ${DESTDIR}${UBINDIR}/mailq_sendmail ${DESTDIR}${UBINDIR}/hoststat ${DESTDIR}${UBINDIR}/purgestat')
 )dnl
 
 # location of sendmail statistics file (usually /etc/mail/ or /var/log)
@@ -72,7 +72,7 @@
 		${LN} ${LNOPTS} ${M`'BINDIR}/sendmail $$i; \
 	done
 
-define(`confMTA_LINKS', `${DESTDIR}${UBINDIR}/newaliases ${DESTDIR}${UBINDIR}/mailq ${DESTDIR}${UBINDIR}/hoststat ${DESTDIR}${UBINDIR}/purgestat')
+define(`confMTA_LINKS', `${DESTDIR}${UBINDIR}/newaliases_sendmail ${DESTDIR}${UBINDIR}/mailq_sendmail ${DESTDIR}${UBINDIR}/hoststat ${DESTDIR}${UBINDIR}/purgestat')
 install-sm-mta: bldCURRENT_PRODUCT
 	${INSTALL} -c -o ${M`'BINOWN} -g ${M`'BINGRP} -m ${M`'BINMODE} bldCURRENT_PRODUCT ${DESTDIR}${M`'BINDIR}/sm-mta
 	for i in confMTA_LINKS; do \
